hbase-server/src/main/java/org/apache/hadoop/hbase/tool/CanaryTool.java

```diff
@@ -1131,8 +1134,8 @@ private void printUsageAndExit() {
     System.err.println(" -f <B>          exit on first error; default=true");
     System.err.println(" -failureAsError treat read/write failure as error");
     System.err.println(" -t <N>          timeout for canary-test run; default=600000ms");
-    System.err.println(" -writeSniffing  enable write sniffing");
-    System.err.println(" -writeTable     the table used for write sniffing; default=hbase:canary");
```

Comment:
I do not think we should remove this flag? People may want to use this flag to specify a different write table for canary